Content.Shared/SubFloor/SubFloorHideComponent.cs

```diff
@@ -26,16 +21,25 @@ public sealed class SubFloorHideComponent : Component
         [DataField("enabled")]
         public bool Enabled { get; set; } = true;
 
+        /// <summary>
+        ///     Whether the entity's current position has a "Floor-type" tile above its current position.
+        /// </summary>
+        public bool IsUnderCover { get; set; } = false;
+
+        /*
+         * An un-anchored hiding entity would require listening to on-move events in case it moves into a sub-floor tile.
+         * Also if this is ever needed, re-examine why `CollideOnAnchorComponent` are ensured????
```

Comment:
Last I checked the pipes still had their collision set even when anchored which is a waste for broadphase when we can just turn it off entirely.